### ISSUE TITLE
Activate/Verification email links not available

### ISSUE THREAD

**Issue by ChaseGiunta** [objectivehtml/Postmaster-for-Craft-CMS#13]
Postmaster does not seem to be sending activation links in the verification emails upon a saveUser event. `{{ user.link }}` is the template code being used (hopefully that's correct).

**Comment by badlydrawnrob** [comment]
Yep @ChaseGiunta, seconded. Sends an empty link. Also, we'd need a way to disable the default Craft email activation right?

    <a href="">postmasterTest</a>

I guess there's no activation link variable for plugins at the moment.

**Comment by badlydrawnrob** [comment]
@ChaseGiunta FYI: http://craftcms.stackexchange.com/a/10694/3888
